### ISSUE TITLE
Why set `affine=False` in all `nn.BatchNorm2d`

### ISSUE THREAD

**Issue by kaleidoscopical** [megvii-model/SinglePathOneShot#3]
I fail to figure out a reasonable explanation. Could anyone help me out?

**Comment by ZichaoGuo** [comment]
We just follow the setting of DARTS.
